cornflow/endpoints/execution.py

```diff
@@ -171,17 +172,28 @@ def get(self, idx):
 
         self.user_id, self.admin, self.super_admin = Auth.return_user_info(request)
         execution = ExecutionModel.get_one_execution_from_user(self.user_id, idx)
-        if execution.finished:
+        if execution.state == 1:
```

Comment:
i think it's better to just return the state as "status" instead of a text. We could also return the `state_message` but the `state` makes more sense.
I mean:
```return dict(status=execution.state), 200```
Alternatively (which I see you have done for the others): we could do:
```return dict(status=execution.state_message), 200```
At least that way we do not duplicate the string with the error.